vlingo-e2e-sys-airline-ops/frontend/src/routes/_layout.svelte

```diff
@@ -22,7 +20,7 @@
 
 <svelte:window on:resize={isMobile.check} />
 
-<div style="height: 100vh; background-color: {bgTheme}">
```

Comment:
Ah yes, I fixed that in materialify so on the other UI's you can delete this as well (or as we said, maybe we make a little component lib)